internal/proto/common.proto

```diff
@@ -183,3 +183,11 @@ enum CompactionState {
   Executing = 1;
   Completed = 2;
 }
+
+enum ConsistencyLevel {
+    Strong = 0;
```

Comment:
Use zero value as a meaningful enum is not recommended. If client `forgot` to set concistency level field, its consistency level becomes `Strong` and not warning or error will be returned.

Reply:
Yeah. But on the other hand, Milvus itself provides the strong consistency by default, maybe it doesn't matter that client forgot to set it. I just thought the consistency level must be set, otherwise an exception will be raised, so I didn't set a meaningless enum to default.
By the way, actually I have no idea which way will be better.

Reply:
I prefer to add a placeholder Invalid = 0; and set Strong =1;

As this [article](https://developers.google.com/protocol-buffers/docs/proto3#default) says,  for scalar message fields, once a message is parsed there's no way of telling whether a field was explicitly set to the default value (for example whether a boolean was set to false) or just not set at all.

Although currently, some Enum definitions in our proto definition do not follow the above rules, it's better to prevent it from appearing in the future.
What do you guys think? @xiaofan-luan @DragonDriver @congqixia